pkg/print/config.go

```diff
@@ -26,3 +27,26 @@ func ConfigValidationErrors(log *logger.CLILogger, groupValidationErrors []confi
 	log.FinishSpinnerWithError()
 	log.Errorf(sb.String())
 }
+
+func PreflightErrors(log *logger.CLILogger, results []*preflight.UploadPreflightResult) {
```

Comment:
why not following the existing pattern of printing? https://github.com/replicatedhq/kots/blob/bc8004b7b2a9d2a14ee1fb03af7c9aba3337ca46/pkg/print/restore.go#L25-L49

that would help align the column titles with the values.

Reply:
> why not following the existing pattern of printing?

No particular reason. I'll try to adopt the same approach